README.md

```diff
@@ -36,6 +36,10 @@ and add your preferred configuration options:
   - Default: `false`
   - Put the `>` of a multiline element on a new line (svelte equivalent of [jsxBracketSameLine](https://prettier.io/docs/en/options.html#jsx-brackets) rule)
 
+- **`svelteIndentScriptAndStyle`**
+  - Default: `true`
+  - Indent the contents of script and style tags inside component files (svelte equivalent of [vueIndentScriptAndStyle](https://prettier.io/docs/en/options.html#vue-files-script-and-style-tags-indentation))
```

Comment:
I'd prefer to skip the reference to the Vue equivalent and instead add a little more explanation.

```suggestion
  - Whether or not to indent the code inside `<script>` and `<style>` tags in Svelte files. This saves an indentation level, but might break code folding in your editor.
```